SPECS/expat/expat.spec

```diff
@@ -67,13 +67,13 @@ rm -rf %{buildroot}/%{_docdir}/%{name}
 %{_libdir}/libexpat.so.1*
 
 %changelog
+* Thu Mar 28 2024 Aditya Dubey <adityadubey@microsoft.com> - 2.6.2-2
+- Updated changelog to match fasttrack chnagelog 
+
 * Thu Mar 21 2024 Aditya Dubey <adityadubey@microsoft.com> - 2.6.2-1
 - Upgrading to 2.6.2 to fix CVE-2023-52425 and CVE-2023-28757
 - No longer need Patch CVE-2023-52426 since 2.6.2 fixes it
```

Comment:
This still doesn't match `fasttrack/2.0`. You can check your changes by running `git diff fasttrack/2.0 HEAD SPECS/expat`.